Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -10,6 +10,12 @@ import Foundation
 import SourceKittenFramework
 
 public struct Configuration: Equatable {
+    // Represents how a Configuration object can be configured with regards to rules.
+    public enum RulesMode {
```

Comment:
I like the idea of this enum, but:
1. Can we think about a better name 🤔 ? (I couldn't)
2. What do you think about making the associated values `Set<String>`?
3. Is there a better name for `default`?